src/components/character-count/threshold/index.njk

```diff
@@ -0,0 +1,17 @@
+---
+title: Character count
+layout: layout-example.njk
+---
+
+{% from "character-count/macro.njk" import govukCharacterCount %}
+
+{{ govukCharacterCount({
+  name: "word-count",
+  id: "word-count",
+  maxlength: 114,
+  threshold: 75,
+  value: "Type another letter into this field after this message to see the threshold feature .",
```

Comment:
@amy.hupe: "Could you update the threshold pre-filled text (Type another letter into this field after this message to see the threshold feature .) to remove the space and the full stop at the end? (And adjust the limit accordingly)". New the limit _should_ be 112, from a rough math.

Reply:
updated